pcsx2/gui/AppCoreThread.cpp

```diff
@@ -376,23 +389,26 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
 				gameMemCardFilter = game.getString("MemCardFilter");
 			}
 
-			if (EmuConfig.EnablePatches) {
-				if (int patches = InitPatches(gameCRC, game)) {
+			if (EmuConfig.EnablePatches)
+			{
+				int patches = InitPatches(gameCRC, game);
+				int fixes = loadGameSettings(fixup, game);
+
+				if (patches)
+				{
 					gamePatch.Printf(L" [%d Patches]", patches);
 					PatchesCon->WriteLn(Color_Green, "(GameDB) Patches Loaded: %d", patches);
 				}
-				if (int fixes = loadGameSettings(fixup, game)) {
+				if (fixes)
 					gameFixes.Printf(L" [%d Fixes]", fixes);
```

Comment:
I don't see how this is better - You're increasing the variable scope beyond what is necessary.